source/assets/styles/_variables.native.ts

```diff
@@ -32,8 +33,6 @@ export const COLORS = {
   gray_medium: '#979797',
   gray_hover: '#dedede',
   gray_disabled: '#dadada',
-  red: '#de0c00',
-  red_100: '#ff3b30',
```

Comment:
Why where these removed? I believe that these are used in some of the on-boarding screens.

Reply:
they're duplicated keys 
the red colors are already listed